tests/Feature/UserTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Tests\Feature;
+
+use Tests\TestCase;
+use App\Models\User;
+use Firebase\JWT\JWT;
+use App\Traits\InteractsWithJWT;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+
+class UserTest extends TestCase
+{
+    use RefreshDatabase, InteractsWithJWT;
+
+    /**
+     * @var User
+     */
+    protected $user;
+
+    /**
+     * Setup the test environment.
+     *
+     * @return void
+     */
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->user = factory(User::class)->create();
+    }
+
+    /** @test */
+    public function guests_cannot_access_the_user_route()
+    {
+        $this->getJson(route('user.index'))
+            ->assertStatus(401);
+    }
+
+    /** @test */
+    public function users_can_access_the_user_route()
+    {
+        $this->actingAs($this->user, 'api')
+            ->getJson(route('user.index'))
+            ->assertStatus(200);
+    }
+
+    /** @test */
+    public function users_can_authenticate_with_a_bearer_token()
+    {
+        $this->getJson(route('user.index'), [
+            'HTTP_Authorization' => 'Bearer ' . $this->createJWT($this->user)
+        ])->assertStatus(200);
```

Comment:
I believe the header string is `Authorization` rather than `HTTP_Authorization`.  I tried running the test locally and it failed, but I think it's unrelated to this header string.

Reply:
`HTTP_Authorization` and `Authorization` both appear to work.  I tried changing the string to something random and watched the test fail just for a sanity check.  I've gone ahead and changed it to `Authorization` sense that appears to be your preference.